lib/std/os/uefi/protocol/serial_io.zig

```diff
@@ -0,0 +1,72 @@
+const std = @import("std");
+const uefi = std.os.uefi;
+const Guid = uefi.Guid;
+const Status = uefi.Status;
+const cc = uefi.cc;
+
+pub const SerialIo = extern struct {
+    revision: u64,
+    _reset: *const fn (*const SerialIo) callconv(cc) Status,
+    _set_attribute: *const fn (*const SerialIo, u64, u32, u32, ParityType, u8, StopBitsType) callconv(cc) Status,
+    _set_control: *const fn (*const SerialIo, u32) callconv(cc) Status,
+    _get_control: *const fn (*const SerialIo, *u32) callconv(cc) Status,
+    _write: *const fn (*const SerialIo, *usize, *anyopaque) callconv(cc) Status,
+    _read: *const fn (*const SerialIo, *usize, *anyopaque) callconv(cc) Status,
+
+    ///Resets the serial device.
+    pub fn reset(self: *const SerialIo) Status {
+        return self._reset(self);
+    }
+
+    ///Sets the baud rate, receive FIFO depth, transmit/receive time out, parity, data bits, and stop bits on a serial device.
+    pub fn set_attribute(self: *const SerialIo, baudRate: u64, receiverFifoDepth: u32, timeout: u32, parity: ParityType, dataBits: u8, stopBits: StopBitsType) Status {
```

Comment:
```suggestion
    pub fn setAttribute(self: *const SerialIo, baudRate: u64, receiverFifoDepth: u32, timeout: u32, parity: ParityType, dataBits: u8, stopBits: StopBitsType) Status {
```
These should be camelCase.